clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -1,99 +1,85 @@
 import { h, FunctionComponent } from "preact";
-import { useState, useEffect } from "preact/hooks";
 import { ButtonType, ExperienceConfig } from "../lib/consent-types";
 import Button from "./Button";
-import "../lib/overlay.module.css";
 import { useHasMounted } from "../lib/hooks";
```

Comment:
let's move the `useHasMounted` from ConsentBanner to Overlay now that Overlay is the parent to all of the other components